pkg/ovirt/helpers.es6

```diff
@@ -59,3 +61,12 @@ export function valueOrDefault(value, def) {
 export function isNumeric(value) {
     return /^\d+$/.test(value);
 }
+
+export function formatDateTime (milliseconds) {
+    const date = new Date(milliseconds);
+    const localeDate = date.toLocaleDateString();
+    const localeTime = date.toLocaleTimeString();
+    const localeString = `${localeDate} ${localeTime}`;
+    logDebug(`formatDateTime: milliseconds=${milliseconds}, localeString: ${localeString}`);
+    return localeString;
+}
```

Comment:
My recommendation of the time format would be:
"Month Number Hour:Minute" (ie. September 17 18:15), Unless the year the VM was started is different from the current year, in that case, the format would be Month Number, Year Hour:Minute (ie. August 12, 2016 20:15).

Reply:
done